.github/workflows/checks.yml

```
@@ -64,6 +64,17 @@ jobs:
         run: |
           go test ./...
 
+  copywrite-tests:
+    name: "License Header Checks"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+      - name: Install copywrite
+        uses: hashicorp/setup-copywrite@v1.1.2
```

Nit: I think we can pin the version here as well for consistency
```suggestion
        uses: hashicorp/setup-copywrite@867a1a2a064a0626db322392806428f7dc59cb3e # v1.1.2
```